_scss/_navigation.scss

```diff
@@ -384,4 +389,9 @@ input:checked+.slider:before {
     margin: 2px 0 0;
 }
 
-
+.divider {
+    height: 1px;
+    margin: 9px 35px;
```

Comment:
Looks like changing this to 29px will make it the same width as the menu items;

```suggestion
    margin: 9px 29px;
```

Before:

<img width="497" alt="Screenshot 2020-05-26 at 13 05 11" src="https://user-images.githubusercontent.com/1804568/82893668-a5242c00-9f51-11ea-9d31-3cf2f6b4bf0f.png">

After:

<img width="505" alt="Screenshot 2020-05-26 at 13 05 01" src="https://user-images.githubusercontent.com/1804568/82893670-a5bcc280-9f51-11ea-90a0-3e3257b66be3.png">